test/helpers/envtest.go

```diff
@@ -91,21 +119,78 @@ type TestEnvironment struct {
 
 	doneMgr chan struct{}
 }
+type WebhookSupportingType interface {
+	SetupWebhookWithManager(ctrl.Manager)
+}
 
 // NewTestEnvironment creates a new environment spinning up a local api-server.
 //
 // This function should be called only once for each package you're running tests within,
 // usually the environment is initialized in a suite_test.go file within a `BeforeSuite` ginkgo block.
 func NewTestEnvironment() *TestEnvironment {
+
+	// initialize webhook here to be able to test the envtest install via webhookOptions
```

Comment:
Might be add a comment explaining that by initializing webhoks it sets `LocalServingCertDir` and `LocalServingPort` that are used below.